.github/workflows/ci.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       - name: Install Rust
         uses: hecrj/setup-rust-action@v1
         with:
-          rust-version: 1.58.0
+          rust-version: 1.58.1
```

Comment:
Is there an action we can use so we don't have to specify this and instead it could look at the toolchain file?

Reply:
There is https://github.com/dtolnay/rust-toolchain

Reply:
Seems to work